### PR TITLE
Command line tool for mapping geometry to the surface of a mesh

### PR DESCRIPTION
**Pull request by TomFischer**
as titled.

I am also open for suggestions to improve the [documentation](http://docs.opengeosys.org/docs/tools/meshing/map-geometric-object-to-the-surface-of-a-mesh) of the tool.

### CODE REVIEW

File: Applications/Utils/MeshEdit/MapGeometryToMeshSurface.cpp [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+		' ', "0.1");
+	TCLAP::ValueArg<std::string> mesh_in("m", "mesh-file",
+		"the name of the file containing the mesh", true,
+		"", "file name");
+	cmd.add(mesh_in);
+	TCLAP::ValueArg<std::string> input_geometry_fname("i", "input-geometry",
+		"the name of the file containing the input geometry", true,
+		"", "file name");
+	cmd.add(input_geometry_fname);
+	TCLAP::ValueArg<bool> advanced_mapping("", "advanced-mapping",
```
Comment (norihiro-w) [inline Applications/Utils/MeshEdit/MapGeometryToMeshSurface.cpp]: can you give a more specific algorithm name here?

**Comment by norihiro-w** [comment]
:+1: except for a minor comment

File: Applications/Utils/MeshEdit/CMakeLists.txt [hunk 2]
```diff
@@ -26,6 +26,20 @@ if(QT4_FOUND)
...
 
+	add_executable(MapGeometryToMeshSurface
+		MapGeometryToMeshSurface.cpp )
+	target_link_libraries(MapGeometryToMeshSurface
+		MeshLib
+		MeshGeoToolsLib
+		FileIO
+		InSituLib
+		${CATALYST_LIBRARIES}
+		${QT_LIBRARIES}
```
Comment (endJunction) [inline Applications/Utils/MeshEdit/CMakeLists.txt]: Are the QT libs necessary?

Comment (TomFischer) [inline Applications/Utils/MeshEdit/CMakeLists.txt]: The catalyst libraries are necessary for the MeshLib. Only the part of Qt used for vtk is necessary. So I removed `${QT_LIBRARIES}`.

**Pull request merged**